plugins/docker/compose_cli.ml

```diff
@@ -1,6 +1,8 @@
 open Lwt.Infix
 
-type t = { pull : bool }
+type t = {
+    pull : bool ;
```

Comment:
```suggestion
  pull : bool;
```

Reply:
Please make them internally consistent in this module, this project seems to use 
`pull: bool;` more consitently than `pull : bool;`